spec/factories/parties.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :party do
+    movie_id { '524' }
+    party_date { Faker::Date.forward }  
+    start_time { "8:00 pm" }  
+    duration { Faker::Number.between(from: 1, to: 500) }  
+    # association :movie
```

Comment:
Added a ticket to relook at this factory (if time) because it's working, but will always create a party for movie 524 right now.